<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,7">
   <item>
    <layout class="QVBoxLayout" name="leftLayout" stretch="1,10">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="topLayout">
       <item>
        <widget class="QComboBox" name="funcChose"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="buttomLayout"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="plotLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
